### DISCUSSION TITLE
Interactivity (callbacks) between plotly charts possible?

### DISCUSSION THREAD

**Discussion by healthfacts** [inseri-swiss/inseri-core-wp#197] [Q&A] [answered]
Hello, i like the inseri-swiss WordPress plugin initiative very much. A question i have: does the inseri-swiss plugin supports Interactivity (callbacks) between plotly charts. If i make two charts based on same data and i select a bar in chart 1 , can chart 2 adapt according the selection?

**Comment by aplamada** [comment, marked as answer]
Hi @healthfacts , this is possible. 
Please have a look at https://inseri.swiss/2023/03/plotly-chart-block/ where by clicking on one bar the corresponding value is displayed. Instead of displaying the value one can add a small code to display the desired chart.

**Reply by healthfacts** [reply]
Thnx Andrei, will give it a try!

**Reply by aplamada** [reply]
You are welcome! If you need help just let me know. Also, let us know your feedback.
